README.md

```diff
@@ -0,0 +1,10 @@
+# MARIAN PCI audio card driver
+
+To compile it, specify the Kernel sources path first:
+```
+export KSRC=<kernel sources path>
```

Comment:
Not the normal way to build out-of-tree modules, but ok...

See my comment on the Makefile change for details.

And kernel code does not need a README.md file :)